### PR TITLE
Adds JDK13 Dockerimage

### PR DESCRIPTION
**Pull request by dh1337**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -59,6 +65,7 @@ publish:
 	./publish.sh --variant alpine ; \
 	./publish.sh --variant slim ; \
 	./publish.sh --variant jdk11 --start-after 2.151 ; \
+	./publish.sh --variant jdk13 --start-after 2.151 ; \
```
Comment (oleg-nenashev) [inline Makefile]: I am not 100% sure about it. Java 11 was in GA after 2.160, and maybe it should be a minimum version here. Or just 2.190 or whatever?

Comment (denishamann1337) [reply Makefile]: My mistake, I updated the version now to the newest

**Comment by MarkEWaite** [comment]
I'm a little skeptical of delivering a docker image for JDK 13 when the [Oracle Java SE roadmap](https://www.oracle.com/technetwork/java/java-se-support-roadmap.html) declares that it is not a long-term support release.

@denishamann1337 are you intending to use this as preparation and testing for an eventual image based on JDK 14 or are you assuming that this image will last beyond the official end of support of JDK 13?  JDK 13 support ends in March 2020, so the JDK used in this image will not receive security fixes or other patches beginning in April 2020.

**Comment by rarspace01** [comment]
Personally aiming for JDK14 LTS is fine by me.
The PR was driven by the fact that in our CI we use Jenkins with the latest JDK. So I wanted to have the option. I can just close the PR

**Comment by arjantijms** [comment]
@MarkEWaite 
> official end of support of JDK 13? 

Technically there is no "official end of support", right? It depends on the vendor of your JDK distribution. The JDK or Java for that matter isn't tied to such support.

**Comment by MarkEWaite** [comment]
> @MarkEWaite
> 
> > official end of support of JDK 13?
> 
> Technically there is no "official end of support", right? It depends on the vendor of your JDK distribution. The JDK or Java for that matter isn't tied to such support.

Agreed that it depends on the vendor of the JDK distribution.

Since Debian only provides JDK 8 and JDK 11, that argument would then recommend that on Debian we either don't use the JDK bundled with Debian, or we don't support any JDK after 11 until Debian supports a JDK after 11.

Since CentOS seems to only provide JDK 8 and JDK 11, if we stay with the CentOS operating system provided package, then we stay with JDK 8 and JDK 11.

If we add or switch to AdoptOpenJDK, then we have their support for JDK 8, JDK 11, and JDK 14.

I haven't looked at the other distributions for their support policies.  The one open proposal we have currently is to switch from JDK provided by the operating system vendor to AdoptOpenJDK.  If that proposal were adopted, then the JDK support would not be more than the JDKs which are currently supported by AdoptOpenJDK.

Can you help me understand what you intended by your comment?

**Comment by slide** [comment]
Conflicts need to be resolved.

**Comment by MarkEWaite** [comment]
Closing this pull request.

The current plan of the Jenkins platform SIG is that new images need to have an assigned maintainer with merge permission to the Jenkins Docker images.  The JDK 13 image proposal does not have a maintainer that is willing to review pull requests or perform other updates.

**Pull request closed without merging**